src/components/AgreementLineSections/Coverage/Coverage.js

```diff
@@ -12,8 +12,6 @@ import {
 
 import { Embargo, isPackage } from '@folio/stripes-erm-components';
 
-import CustomCoverageIcon from '../CustomCoverageIcon';
```

Comment:
Was removing the CustomCoverageIcon part of some JIRA?

Reply:
No, but it was causing the test to fail, we discussed that with Gill and Ethan, and they suggested that to remove it as it was not necessary to have the icon next to the custom coverage content.

Reply:
Ah okay, sounds good.